static/js/event/event.ts

```diff
@@ -21,11 +21,18 @@
 import React from "react";
 import ReactDOM from "react-dom";
 
-import { Page } from "./page";
+import { EventPage } from "./page";
 
 window.onload = () => {
```

Comment:
When creating **new** pages, types, etc, please follow the file structure outlined [here](https://docs.google.com/document/d/1eYvx8sYVNxJ5tWQ229gHb3lPCdepwBt5Z2TYKRdvSNQ/edit?resourcekey=0-_Sz3NjUjbJ621ZtVB9PD6Q#heading=h.95wucwdioq2) (trying to gradually update the whole repo to this structure). An example that uses this structure is the new [disaster dashboard](https://github.com/datacommonsorg/website/tree/master/static/js/apps/disaster_dashboard).

Reply:
Moved to follow the new structure